admin/bin/qserv-configure.py

```diff
@@ -85,10 +85,11 @@ def __init__(self):
                    "  - Create an execution directory (QSERV_RUN_DIR) which contains configuration and"
                    "execution\n"
                    "    data for a given Qserv instance.\n"
-                   "  - Create a data directory (QSERV_DATA_DIR) which contains MySQL/Qserv/Zookeeper data.\n"
-                   "  - Use templates and meta-config file parameters (see QSERV_RUN_DIR/qserv-meta.conf) to"
+                   "  - Create a data directory (QSERV_DATA_DIR) which contains MySQL/Qserv/Zookeeper data,\n"
+                   "QSERV_RUN_DIR/var/lib symlink to QSERV_DATA_DIR if different"
```

Comment:
Please indent this like and add newline at the end.
You should have run qserv-configure from your branch, you'd see all these formatting issues.